src/dom/Value.ts

```diff
@@ -206,18 +206,21 @@ export interface Value {
    *        ignoreAnnotations           specifies whether to ignore annotations or not for equality. (Default: false)
    *        ignoreTimestampPrecision    specifies whether to ignore timestamp local offset and precision
    *                                    or not for equality. (Default: false)
-   *        onlyCompareIon:             specifies if only dom.Values should be considered for equality
-   *                                    or can be JS object as well. (Default: true)
    */
-  equals(
+  ionEquals(
     other: any,
     options?: {
       epsilon?: number | null;
       ignoreAnnotations?: boolean;
       ignoreTimestampPrecision?: boolean;
-      onlyCompareIon?: boolean;
     }
   ): boolean;
+
+  /**
+   * For checking non-strict equivalence of an Ion Values represented by dom.Value with the other value represented
```

Comment:
```suggestion
   * Compares this instance of dom.Value to the provided value and returns true
   * if they are equal. If `other` is a dom.Value, this comparison checks for structural
   * (or "non-strict") equivalence. If `other` is a native JS value, this comparison will
   * convert it to an Ion value and then check for structural equivalence.
```

Reply:
Sorry, I just re-read the implementation for `_valueEquals` and remembered it doesn't convert JS types to Ion types during the comparison. That part of my suggested edit wouldn't be accurate.